daemon/images.go

```diff
@@ -96,7 +96,7 @@ func (r resources) IDs() (ids []flux.ResourceID) {
 }
 
 // getUnlockedAutomatedServices returns all the resources that are
-// both automated, and not locked.
+// both automated, and not locked or ignored.
```

Comment:
Can you rephrase it so that the reader doesn't need to think about logic operator precedence?